src/Core/Kernel.php

```diff
@@ -49,29 +48,14 @@ private function prepareContainer()
         if (!$this->container) {
             $builder = new ContainerBuilder(new ParameterBag());
             $builder->addCompilerPass(new RegisterListenersPass());
-            $definition = new Definition(ContainerAwareEventDispatcher::class, [new Reference('service_container')]);
+            $definition = new Definition(EventDispatcher::class, [new Reference('service_container')]);
+            $definition->setPublic(true);
```

Comment:
hi @adunsulag ,
The above 2 lines got things working for symfony 4.4 . Just wanted to make sure I am not doing anything idiotic here :)

Reply:
No this is the correct way of doing it.

Reply:
awesome. thanks for checking it out.